packaging_automation/update_docker.py

```diff
@@ -169,9 +163,14 @@ def read_postgres_versions(pkgvars_file: str) -> Tuple[str, str, str]:
     os.chdir(execution_path)
     pr_branch = f"release-{args.prj_ver}-{uuid.uuid4()}"
     run(f"git checkout -b {pr_branch}")
+
     update_all_docker_files(args.prj_ver, execution_path)
     run("git add .")
 
+    # git add command tries to add all the files, including cloned tools repo as submodule which is not needed
+    # this command removes the submodule from the commit
+    run("git rm -f --cached tools")
```

Comment:
```suggestion
    run("git add --update .")
```

This command adds only those files that are already in the git worktree.